From 1a2a689deacaac32b351ae97b00d8c35a6499cf6 Mon Sep 17 00:00:00 2001
From: Xavier Claessens <xavier.claessens@collabora.com>
Date: Sat, 29 Nov 2014 08:39:24 -0500
Subject: Doc: glib: Fix all undocumented/unused/undeclared symbols

There is one issue left in gscanner.h due to a bug #741305 in gtk-doc.

https://bugzilla.gnome.org/show_bug.cgi?id=740814
---
 glib/gunicode.h                        |  4 +--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/glib/gunicode.h b/glib/gunicode.h
index 7ec6862..3109233 100644
--- a/glib/gunicode.h
+++ b/glib/gunicode.h
@@ -259,7 +259,7 @@ typedef enum
  *                               a value never returned from g_unichar_get_script()
  * @G_UNICODE_SCRIPT_COMMON:     a character used by multiple different scripts
  * @G_UNICODE_SCRIPT_INHERITED:  a mark glyph that takes its script from the
- * i                             base glyph to which it is attached
+ *                               base glyph to which it is attached
  * @G_UNICODE_SCRIPT_ARABIC:     Arabic
  * @G_UNICODE_SCRIPT_ARMENIAN:   Armenian
  * @G_UNICODE_SCRIPT_BENGALI:    Bengali
@@ -393,7 +393,7 @@ typedef enum
  * @G_UNICODE_SCRIPT_PSALTER_PAHLAVI:      Psalter Pahlavi. Since: 2.42
  * @G_UNICODE_SCRIPT_SIDDHAM:              Siddham. Since: 2.42
  * @G_UNICODE_SCRIPT_TIRHUTA:              Tirhuta. Since: 2.42
- * @G_UNICODE_SCRIPT_WARANG_CITI           Warang Citi. Since: 2.42
+ * @G_UNICODE_SCRIPT_WARANG_CITI:          Warang Citi. Since: 2.42
  *
  * The #GUnicodeScript enumeration identifies different writing
  * systems. The values correspond to the names as defined in the
-- 
cgit v0.12

